routes/caniuse/index.js

```diff
@@ -40,7 +41,6 @@ async function route(req, res) {
     return;
   }
 
-  // cache for 24hours (86400 seconds)
-  res.set("Cache-Control", "max-age=86400");
+  res.set("Cache-Control", `max-age=${seconds("24h")}`);
```

Comment:
I don't have a strong opinion, but would the following be nicer?
``` js
  res.set("Cache-Control", `max-age=${seconds`24h`}`);
```

Reply:
Function is fine.